### ISSUE TITLE
Problem about the val loss

### ISSUE THREAD

**Issue by xym2009** [xsxjtu/densefuse_pytorch#1]
Thanks for the Pytorch code!

In line 85 'tmp2 += (SSIM_fun(img,img_recon)*img.shape[0]).item()', 
the way to calculate the ssim loss of the val dataset should be 1-SSIM_fun(img,img_recon)?
In line 86-89 
'mse_val.append(tmp1/data_val.__len__())
ssim_val.append(tmp1/data_val.__len__())                 
loss_val.append(tmp1/data_val.__len__())',  
the mse, ssim and loss all utilize the tmp1 result, is that correct?
